<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlgorithmControlWidget</class>
 <widget class="QWidget" name="AlgorithmControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Algorithm Control Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frameVideo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QFrame" name="frameControls">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="checkVideoStab">
        <property name="text">
         <string>Video stabilization</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkTamper">
        <property name="text">
         <string>Tamper detection</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkMasking">
        <property name="text">
         <string>Privacy masking</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Preprocessing</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QRadioButton" name="radioPre_1">
           <property name="text">
            <string>None</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioPre_2">
           <property name="text">
            <string>Contrast enhancement</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioPre_3">
           <property name="text">
            <string>Dehazing</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QScrollBar" name="horizontalScrollBarPre">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>50</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboRunMode">
        <item>
         <property name="text">
          <string>Auto Track</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Semi Auto Track</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Manuel Track</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Smart Motion</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bypass</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>223</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
